src/chatClient.cpp

```diff
@@ -3602,13 +3659,23 @@ bool GroupChatRoom::syncMembers(const mega::MegaTextChat& chat)
         }
     }
 
+    if (commitEach)
+    {
+        parent.mKarereClient.setCommitMode(true);
+    }
```

Comment:
No need to check previous state. If it has not changed, it does nothing.
```suggestion cpp
        parent.mKarereClient.setCommitMode(commitEach);
```

Reply:
Yes, but maybe we want to keep in commit mode false, that is why I'm checking the previous state. The commit mode keep in commit mode false while I'm not connected to all chats. In that point, I don't know if I'm connected to all chats or I have to receive more info from other chats